x-pack/metricbeat/module/aws/sqs/_meta/fields.yml

```diff
@@ -37,6 +37,6 @@
       description: >
         The number of ReceiveMessage API calls that did not return a message.
     - name: sent_message_size.bytes
-      type: scaled_float
+      type: long
```

Comment:
Just realised here we miss `format: bytes`.

Reply:
Done! Thanks!